assets/js/blocks/cart/inner-blocks/cart-items-block/edit.tsx

```diff
@@ -13,11 +14,18 @@ import {
 	getAllowedBlocks,
 } from '../../../cart-checkout-shared';
 
-export const Edit = ( { clientId }: { clientId: string } ): JSX.Element => {
+interface Props {
+	clientId: string;
+}
+
+export const Edit = ( { clientId }: Props ): JSX.Element => {
 	const blockProps = useBlockProps( { className: 'wc-block-cart__main' } );
 	const allowedBlocks = getAllowedBlocks( innerBlockAreas.CART_ITEMS );
 	const defaultTemplate = [
 		[ 'woocommerce/cart-line-items-block', {}, [] ],
+		...( isExperimentalBuild()
+			? [ 'woocommerce/cart-cross-sells-block', {}, [] ]
+			: [] ),
```

Comment:
Hi @nielslange! You have reported in Slack the following bug: `TypeError: t is not iterable at blocks.min.js?`.  The fix is simple! You just need to get rid of the spread operator in this expression `...`:

```js
const defaultTemplate = [
		[ 'woocommerce/cart-line-items-block', {}, [] ],
		isExperimentalBuild()
			? [ 'woocommerce/cart-cross-sells-block', {}, [] ]
			: [ '' ],
	] as TemplateArray;
```

Or, you can simply affect this expression to a variable first to make it more readable:

```js

const cartCrossSellsTemplate = isExperimentalBuild()
			? [ 'woocommerce/cart-cross-sells-block', {}, [] ]
			: [ '' ];
const defaultTemplate = [
		[ 'woocommerce/cart-line-items-block', {}, [] ],
		cartCrossSellsTemplate,
	] as TemplateArray;
```

Reply:
Thanks a lot, @tarhi-saad! The fix worked like charm! 🙌